### DISCUSSION TITLE
Issue with setting strokeWidth in components within a Group

### DISCUSSION THREAD

**Discussion by lukaszkowalik2** [fabricjs/fabric.js#9738] [Q&A] [answered]
I am encountering an issue with setting the strokeWidth property in Fabric.js components such as Rect, Circle, Ellipse, and Polygon when they are placed within a Group. In my example, changing the strokeWidth and clicking on the object results in the bounding box (bbox) not updating correctly, requiring the use of rect.add() to re-render the borders. This issue seems to be specific to components within a Group context.

Link to example: [Link to CodeSandbox](https://codesandbox.io/p/devbox/bug-3zyww7?file=%2Fsrc%2FApp.tsx%3A58%2C5)

Your insights or suggestions regarding this behavior of Fabric.js components within a Group would be highly appreciated. Thank you for your assistance!

**Comment by asturur** [comment]
I can't open the sandbox.
Which version of fabricJS are you using?

<img width="602" alt="image" src="https://github.com/fabricjs/fabric.js/assets/1194048/1bb7f2b1-5931-46d8-987a-04cc18488eb6">

If you are using fabric v6 try to use group.performLayout() after you change the strokeWidth.

If you are using fabric 5 i think you have either to:
call group.addWIthUpdate() without objects or remove and add back the object

**Comment by ShaMan123** [comment, marked as answer]
You need to trigger layout on the group
https://codesandbox.io/p/live/073dc6bd-57aa-40e1-8d42-8f354558d293
